plugin/evm/cotracts_test.go

```diff
@@ -0,0 +1,133 @@
+//go:build integration
```

Comment:
Careful, the name of this file has a typo in it. Also I guess it should have a copyright header? Not sure if they're 100% consistent with their headers in coreth.

Reply:
Yeah, I'm not sure about the headers either